app/controllers/api/v2/filters_controller.rb

```diff
@@ -35,7 +35,7 @@ def destroy
   private
 
   def set_filters
-    @filters = current_account.custom_filters.includes(:keywords)
+    @filters = CustomFilter.includes(:keywords).where(account_id: [current_account.id, CustomFilter.instance_filter.account_id])
```

Comment:
I'm somewhat uncomfortable exposing the instance wide filter this way, but not doing so results in the filtered attribute on toots to be removed when filters are fetched in the web UI (by opening the filter modal for example).
This specific thing is fixed by using the `hidden_by_moderator` attribute being used in the web UI instead, but I have no way to test how this behaves in apps.
And only logged in users have access to this information.

Reply:
Not doing this seems to cause inconsistent behaviour where toots are being displayed as hidden and unhidden inconsistently.